src/lib.rs

```diff
@@ -324,6 +324,32 @@
 //! # }
 //! ```
 //!
+//! To retrieve ranges from a versioned object, use [`ObjectStore::get_opts`] by specifying the range in the [`GetOptions`].
+//!
+//! ```ignore-wasm32
```

Comment:
I am not familiar with this `ignore-wasm32` annotation -- it seems like it may just not run these examples at all

If we are having trouble running the doc tests on wasm32, I suggest we just don't run the doc examples on wasm32 in CI